test/test_grizzly_ber.rb

```diff
@@ -230,11 +230,12 @@ def test_null_tags_are_bad
   end
 
   def test_cyrillic_application_name_doesnt_crash
-    tlv = GrizzlyBer.new "9F0608A0000000031010048408A0000000031010049F1C08313030303332373"\
-              "89F1E0831303030333237389F120CB2D8E1D020B4D5D1D5E22032500C56495341204445424"\
-              "95420329F34031E0300950502000080009B02E8009F100706010A03A000009F3303E0B8C88"\
-              "2025C009F0902008C9A03150317139593"
+    tlv = GrizzlyBer.new "9F120CB2D8E1D020B4D5D1D5E22032"
```

Comment:
The unit test already expects nothing to ever be raised and assert_nothing_raised is a NOP:
http://stackoverflow.com/questions/12499259/why-doesnt-minitestspec-have-a-wont-raise-assertion